d2l-dom-focus.html

```diff
@@ -195,19 +195,49 @@
 
 		},
 
-		isFocusable: function(node, includeHidden) {
+		getPreviousFocusableAncestor: function(node, includeHidden, includeTabbablesOnly) {
+
+			if (!node) {
+				return null;
+			}
+
+			if (includeHidden === undefined) {
```

Comment:
The reason I ask is because `includeHidden` does explicitly set default, vs `includeTabbablesOnly` does not.  I think I would probably be consistent in the approach.